pallets/stake/src/lib.rs

```diff
@@ -1092,10 +1161,7 @@ where
 	}
 }
 
-impl<T> author_inherent::CanAuthor<T::AccountId> for Module<T>
-where
-	T: Config + author_inherent::Config,
-{
+impl<T: Config + author_inherent::Config> author_inherent::CanAuthor<T::AccountId> for Module<T> {
```

Comment:
```suggestion
impl<T: Config> author_inherent::CanAuthor<T::AccountId> for Module<T> {
```

Same. But double check me on this.